### ISSUE TITLE
list of routes

### ISSUE THREAD

**Issue by francescoagati** [haxetink/tink_web#132]
hello,
there is a mode for have a list of all the route generated with also response?

**Comment by kevinresol** [comment]
Nope. There was an attempt to generate swagger specs, but it was never finished.
